doc/examples/applications/plot_solidification_tracking.py

```diff
@@ -0,0 +1,279 @@
+
+fig = px.imshow(
+    denoised,
+    animation_frame=0,
+    binary_string=True,
+    labels={'animation_frame': 'time point'}
+)
+plotly.io.show(fig)
+
+#####################################################################
+# Binarize
+# ========
+# Our next step is to create binary images, splitting the images
+# into foreground and background: We want the S-L interface
+# to be the most prominent feature in the foreground of each binary image,
+# so that it can eventually be separated from the rest of the image.
+#
+# We need
+# a threshold value ``thresh_val`` to create our binary images, ``binarized``.
+# This can be set manually, but we shall use an automated minimum threshold
+# method from the ``filters`` submodule of scikit-image (there are other
+# methods that may work better for different applications).
+
+thresh_val = filters.threshold_minimum(denoised)
+binarized = denoised > thresh_val
+
+fig = px.imshow(
+    binarized,
+    animation_frame=0,
+    binary_string=True,
+    labels={'animation_frame': 'time point'}
+)
+plotly.io.show(fig)
+
+#####################################################################
+# Select largest region
+# =====================
+# In our binary images, the S-L interface appears as the largest region of
+# connected pixels. For this step of the workflow, we will operate on each
+# 2D image separately, as opposed to the entire 3D dataset. We are
+# only interested in regions connected in space, so we operate
+# on one image at a time so the regions do not span multiple moments in
+# time across multiple images. We do this with a list comprehension by labeling each binarized
+# image with the function :func:`skimage.measure.label()` while slicing the dataset from
+# ``i = 0`` to ``i = binarized.shape[0]``.
+# We can do this first labeling each separate
+# region of connected pixels in the binary images
+# The labels can be visualized by overlaying
+# ``labeled`` over the original images (``image_sequence``). The function
+# :func:`skimage.color.label2rgb()` takes a 2D image, so we must be
+# careful to account for offset introduced after the image delta step when
+# plotting these images.
+
+labeled_list = [
+        measure.label(binarized[i, :, :]) for i in range(binarized.shape[0])]
+labeled_overlay_0 = color.label2rgb(
+        labeled_list[0], image=image_sequence[1, :, :], bg_label=0)
+
+fig = px.imshow(labeled_overlay_0, color_continuous_scale='gray')
+plotly.io.show(fig)
+
+#####################################################################
+# We will now select the largest region in each image. We can do this
+# by computing region properties, including the ``area`` property, and
+# sorting by ``area`` values. Function
+# :func:`skimage.measure.regionprops_table()` returns a table of region
+# properties which can be readily read into a Pandas ``DataFrame``.
+
+props_0 = measure.regionprops_table(
+        labeled_list[0], properties=('label', 'area', 'bbox'))
```

Comment:
```suggestion
        measure.label(binarized[0, :, :]), properties=('label', 'area', 'bbox'))
```
Yes, let's show this processing step on a single 2D image first (we should add a line about this in the text). But, at this stage, we don't need to overlay the regions obtained from the image deltas with the original images; we are focusing on finding the largest region on each 2D image.